package.json

```diff
@@ -66,21 +65,27 @@
     "@opentelemetry/semantic-conventions": "1.22.0",
     "ajv": "8.13.0",
     "commander": "12.0.0",
-    "drizzle-orm": "^0.32.2",
-    "inquirer": "^8.2.6",
     "kafkajs": "^2.2.4",
     "knex": "3.1.0",
     "koa": "2.15.0",
     "lodash": "4.17.21",
     "openapi-types": "12.1.3",
     "pg": "8.11.3",
+    "pg-protocol": "^1.6.1",
```

Comment:
What is this?

Reply:
This is used in dbos-executor.ts:
```
import { NoticeMessage } from "pg-protocol/dist/messages";
```

A missing dependency found by depcheck. It is a dependency of `pg` but we should make it explicit since we're using it.